### PR TITLE
feat!: update vrs/common models

### LINKED ISSUE #409: Implement CisPhasedBlock
And delete Haplotype.

https://github.com/ga4gh/vrs/blob/2.x/schema/vrs/vrs-source.yaml#L148

### PR DESCRIPTION
**Pull request by korikuzma**
close #409 , #116

* updates vrs/gks-common submodules
  * Pydantic models updated to reflect these changes
*  Removed `use_enum_values` from Pydantic config. We didn't use this in all models, so not sure if there was a purpose for it.
* Removed duplicate vrs test

Could definitely get some other eyes on this to make sure I didn't miss anything

### CODE REVIEW

File: src/ga4gh/core/_internal/models.py
Comment (korikuzma) [inline src/ga4gh/core/_internal/models.py:1]: Wondering if we should split these out into core-im vs domain entity modules?

File: tests/test_vrs2.py
Comment (korikuzma) [inline tests/test_vrs2.py:1]: I deleted test_vrs.py and renamed test_vrs2.py to test_vrs.py. These were essentially the same, but test_vrs2.py had more tests.

**Comment by korikuzma** [comment]
Going to do one more look over in the morning. Then I think I'm ready to mark ready for review.

File: src/ga4gh/core/_internal/models.py
Comment (korikuzma) [reply src/ga4gh/core/_internal/models.py:1]: If so, might do this in a separate PR

**Review by jsstevenson — APPROVED** [review]
looks good to me*

*someone who is not particularly involved in any of the changes to these models

**Pull request merged**